pyignite/utils.py

```diff
@@ -225,12 +225,13 @@ def datetime_hashcode(value: int) -> int:
     return (value & LONG_MASK) ^ (unsigned(value, ctypes.c_ulonglong) >> 32)
 
 
-def status_to_exception(exc: Type[Exception]):
+def status_to_exception(exc: Type[Exception], ignore_timeout=False):
```

Comment:
Maybe timeout should be mentioned in the name now? Not sure though.

Reply:
Agree, that name is not great. But I haven't found the better variant though :(